### PR TITLE
TN-1108 initial queries refinements

### PR DESCRIPTION
**Pull request by achen2401**
- remove unused argument from getRoles method, remove unused method in main.js
- move document ready block from initial queries template to js file

### CODE REVIEW

File: portal/static/js/initialQueries.js [hunk 1]
```diff
@@ -30,10 +8,11 @@
                 throw new Error("Dependency with key value: " + key + " not found.");
             }
         };
-        this.userId = userId;
+        this.userId = 0;
```
Comment (ivan-c) [inline portal/static/js/initialQueries.js]: I might use a non-numeric value (`null`?) if it hasn't been initialized to make it easier to distinguish...

File: portal/static/js/initialQueries.js [hunk 2]
```diff
@@ -1005,4 +1012,44 @@
...
+
+    $(document).ready(function() {
+        /*
+         * the flow here :
+         * get still needed core data
+         * populate all required fields
+         * get incomplete fields thereafter
+         * note: need to delay gathering incomplete fields to allow fields to be render
+         */
+        if ($("#aboutForm").lenght > 0 || $("#topTerms").length > 0) { /*global i18next, tnthAjax, OrgTool, tnthDates*/
```
Comment (ivan-c) [inline portal/static/js/initialQueries.js]: >lenght

typo?

File: portal/static/js/initialQueries.js [hunk 3]
```diff
@@ -1005,4 +1012,44 @@
...
+
+    $(document).ready(function() {
+        /*
+         * the flow here :
+         * get still needed core data
+         * populate all required fields
+         * get incomplete fields thereafter
+         * note: need to delay gathering incomplete fields to allow fields to be render
+         */
+        if ($("#aboutForm").lenghth > 0 || $("#topTerms").length > 0) { /*global i18next, tnthAjax, OrgTool, tnthDates*/
```
Comment (ivan-c) [inline portal/static/js/initialQueries.js]: >lenghth 

one more time? :D

**Comment by achen2401** [comment]
@ivan-c Ivan, I am having issue typing today :-O

**Pull request merged**